### DISCUSSION TITLE
Question on license change

### DISCUSSION THREAD

**Discussion by rmasonhx12** [GrapesJS/grapesjs#5738] [Q&A]
I understand that Grapejs is based on BSD but while researching found another tool called silex which seems to have changed the license from BSD to AGPL 3.0 license --https://github.com/silexlabs/Silex -- which if I understand open source correctly this change would have to be agreed by you plus all contributors and anyone else who has copy of grapejs for developmentHere are the main features of Silex website builder:

Free and Open Source, open to contributions
Visual Editor: Silex offers a visual editor that allows users to create websites without needing to write code. It supports drag-and-drop functionality for easy website creation.
Online CSS Editor: Alongside the visual editor, Silex also provides an online CSS editor for more advanced customization.
Static HTML Websites: Silex is designed to create static HTML websites, which are fast, secure, and easy to host.
SEO Features: Silex includes SEO features to help improve the visibility of websites on search engines.
Editor UI: Silex offers features like fonts, visual CSS editor, publication...
Self-hosting Option: Users can choose to host their Silex website builder on their own server.
Community Support: Being an open-source project, Silex has a community of developers who contribute to its development and provide support.
Silex v3 > Silex is based on [GrapesJs](https://grapesjs.com/)
Silex v2 > Sync with Dropbox and FTP: Users can sync their Silex projects with Dropbox and FTP, allowing for easy access and management of files.
Silex v2 > Templates: Silex comes with a growing number of templates (both free and paid) that users can use as a starting point for their website

**Comment by lexoyo** [comment]
Thank you for your question and interest in Silex. Here’s an explanation:

Silex uses GrapesJS as its framework, which is licensed under the BSD 3-Clause license. This license is permissive and compatible with Silex's AGPL-3.0 license. The two projects are separate projects.

Side note: there is no contributor agreement in place for Silex because I believe that contributions should remain the property of their original authors. This approach aligns with my values, as I don’t think contributors should have to give up their rights to their work. Instead, by contributing, authors are sharing their code under the same AGPL license, ensuring everyone benefits under the same terms.

If you have further questions about the licensing or technical aspects of Silex, I’m happy to dive deeper!
